freedomofpress/securedrop
# Update pylint, astroid, mypy

## Status

Ready for review 

## Description of Changes

Update pylint to get a new version of astroid so that we can lint under Python 3.7. Update mypy to update typed-ast to a version compatible with the new version of astroid. Fix an obscure string
formatting error in a test assertion that was caught by the new mypy. 

Fixes #5132.

## Testing

- Check out this branch.
- Remove and recreate your virtualenv: `rm -rf .venv; make venv`
- Run `make lint`. It should complete with no errors.

## Deployment

Dev only.

## Checklist

### If you made changes to the server application code:


### If you made non-trivial code changes:


### If you added or updated a code dependency:

Choose one of the following:

- [x] It was only a development dependency.
